src/components/StarRatingContainer.ts

```diff
@@ -0,0 +1,127 @@
+import { Component, createElement } from "react";
+import { StarRating } from "./StarRating";
+
+interface WrapperProps {
+    class?: string;
+    mxObject?: mendix.lib.MxObject;
+    style?: string;
+}
+
+export interface ContainerProps extends WrapperProps {
+    editable: "default" | "never";
+    mxObject: mendix.lib.MxObject;
+    viewAverage: boolean;
+    readOnly: boolean;
```

Comment:
`readOnly `and `mxObject` are redundant, as they are already extended from `WrapperProps`, please remove